### PR TITLE
adding files for NASA assignment

### PR DESCRIPTION
**Pull request by evantwidwell**
See pull request for NASA assingment

### CODE REVIEW

File: src/checkpoint1/checkpoint1.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>calc-flex</title>
+  <link rel="stylesheet" href="../../src/reset.css">
```
Comment (juan-rod) [inline src/checkpoint1/checkpoint1.html]: `<link rel="stylesheet" href="../../reset.css">`
change to this. reset.css is not in the src folder

File: src/flexbox/nasa.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+  <html lang="en">
+  <head>  
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="../../src/reset.css">
```
Comment (juan-rod) [inline src/flexbox/nasa.html]: same as above:
<link rel="stylesheet" href="../../reset.css">
change to this. reset.css is not in the src folder

File: src/flexbox/nasa.css [hunk 3]
```diff
@@ -0,0 +1,133 @@
+body{
+  display:flex;
+  flex-direction: column;
+  height: 800px;
+  width: 600px;
+
```
Comment (juan-rod) [inline src/flexbox/nasa.css]: i'm not a big fan of setting a specific height/width on body. these should be 100% you should wrap the `<div class="image">` and `<div class="footer">` elements with a `<div class="main-container">`. and set the width and height on that.

File: src/flexbox/nasa.css [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+  background-size: cover;
+  height: 60%;
+  width: 600px;
```
Comment (juan-rod) [inline src/flexbox/nasa.css]: set this as 100%;. full width of page.

File: src/flexbox/nasa.css [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+}
+.image h1{
+  font-size: 30px;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 80px;
+}
+.footer{
+  height: 40%;
+  width: 600px;
```
Comment (juan-rod) [inline src/flexbox/nasa.css]: same for this. 100%. full width of page

File: src/flexbox/nasa.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    <link rel="stylesheet" href="./nasa.css">
+    </head>
+    <body>
+
```
Comment (juan-rod) [inline src/flexbox/nasa.html]: remove unnecessary spaces

**Review by juan-rod — APPROVED** [review]
great work! definitely improved! make some changes. push them back up and hit that green merge button!

**Review by evantwidwell — COMMENTED** [review]
Hey just trying to resend this. I know there was an issue with me actually sending the whole checkpoint file on the same PR

**Pull request merged**